src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -143,14 +113,15 @@ case class ConflictingEntities(conflicts: Seq[String])
 
 trait Attribute
```

Comment:
Can make these traits `sealed` to ensure that they are only extended in this file